jcommune-plugins/questions-n-answers-plugin/src/main/java/org/jtalks/jcommune/plugin/questionsandanswers/controller/QuestionsAndAnswersController.java

```diff
@@ -115,12 +111,23 @@ public String showNewQuestionPage(@RequestParam(BRANCH_ID) Long branchId, Model
             throws NotFoundException {
         VelocityEngine engine = new VelocityEngine(getProperties());
         engine.init();
+
+        TopicDto dto;
+        TopicDraft draft = getPluginTopicDraftService().getDraft();
+
+        if (draft != null) {
```

Comment:
Can be simplified with a small trick: 
`TopicDraft draft = ObjectUtils.defaultIfNull(getPluginTopicDraftService().getDraft(), new TopicDraft())`